less/index.less

```diff
@@ -115,62 +122,81 @@ header .logo {
     flex-flow: column;
     align-items: center;
     margin: 0 82px 0 0;
-}
-
-.cta .cta-text button {
-    border: 1px double black;
-    font-size: 16px;
-    background: white;
-    width: 170px;
-    height: 32px;
-    margin-top: 10px;
-    cursor: pointer;
-}
-
-.cta .cta-text button:hover {
-    background: black;
-    color: white;
-}
-
-/* Main Content Styles */
+    @media (max-width: 795 + 5 px)
+    {
+      color: aquamarine;
+    }
+    button {
+      color: @not-so-primary-colour;
+      border: 1px double @not-so-primary-colour;
+      font-size: 16px;
+      background: white;
+      width: 170px;
+      height: 32px;
+      margin-top: 10px;
+      cursor: pointer;
+      &:hover {
+        background: black;
+        color: white;
+      }
+    }
+  }
+}
+
+// Main Content Styles
 
 .main-content {
-   border-top: 2px solid black; 
-   border-bottom: 2px solid black;
-   margin: 75px 0 35px 0;
-   padding: 35px 0;
-   font-size: 16px;
-   line-height: 18px;
-}
-
-.main-content .top-content {
+  border-top: 2px solid black;
+  border-bottom: 2px solid black;
+  margin: 75px 0 35px 0;
+  padding: 35px 0;
+  font-size: 16px;
+  line-height: 18px;
+  p {
```

Comment:
Your `p` isn't inheriting `font-size` from `.main-content` at your breakpoint. So I would assign a `font-size` in the media query itself